assignment2/jsondb/builds.json

```diff
@@ -0,0 +1,6 @@
+{"schemaVersion":"1.0"}
+{"sha":"0a1e0bc0c64d1a1a32d325888f4386995edde265","start_time":"2021-02-17T00:35:35.254948","compile_success":true,"test_success":true,"tests_run":1,"tests_failed":0,"tests_errors":0}
+{"sha":"eacb7513de2a0810024a83b04c0f302bd0eb2c89","start_time":"2021-02-17T00:52:47.773515","compile_success":true,"test_success":true,"tests_run":1,"tests_failed":0,"tests_errors":0}
+{"sha":"4e94a8c9b9e78c74dd07fe60f236b56780c3319d","start_time":"2021-02-17T00:53:42.548151","compile_success":false,"test_success":false,"tests_run":0,"tests_failed":0,"tests_errors":0}
+{"sha":"14611225dcbce2d58705c42aec18c30a1a6e1347","start_time":"2021-02-17T00:54:22.563884","compile_success":true,"test_success":true,"tests_run":1,"tests_failed":0,"tests_errors":0}
+{"sha":"d6acdc80f3b8d811a8fb97b2c2ae441773fce0ff","start_time":"2021-02-17T00:55:09.372682","compile_success":true,"test_success":false,"tests_run":1,"tests_failed":1,"tests_errors":0}
```

Comment:
I don't think we should push this database because the new version has API changes